finalPrep/modZeroHero.js

```diff
@@ -1,40 +1,60 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
-
+var heroName = "Water Girl";
+var specialAbility = "controls all bodies of water with her mind";
 // Declare two variables - greeting AND catchphrase
 //   greeting should be assigned to a string that uses concatenation to include the heroName
 //   catchphrase should be assigned to a string that uses interpolation to include the specialAbility
-
+var greeting = "Hi I'm Water Girl";
+var catchphrase = "I'm here to save the day";
 // Declare two variables - power AND energy - set to integers
-
+var power = 9000;
+var energy = -5;
 // Declare two variables - fullPower AND fullEnergy
 //   fullPower should multiply your current power by 500
 //   fullEnergy should add 150 to your current energy
-
+var fullPower = 10000000;
+var fullEnergy = 2000000;
 // Declare two variables - isHuman and identityConcealed - assigned to booleans
-
+var isHuman = true;
+var identityConcealed = true;
 
 // Declare two variables - archEnemies AND sidekicks
 //   archEnemies should be an array of at least 3 different enemy strings
 //   sidekicks should be an array of at least 3 different sidekick strings
-
+var archEnemies = ["Crazy Stalker Girl", "Ugly Ex Boyfriend", "Bully from Middle School"];
+var sideKick = ["Mia the Shepard", "Rudy the Water Monster", "Ariel the Mermaid"];
 // Print the first sidekick to your console
-
+console.log(sideKick[0]);
 // Print the last archEnemy to the console
-
+console.log(archEnemies[2]);
 // Write some code to add a new archEnemy to the archEnemies array
-
+archEnemies[3] = "Scary Gorilla"
 // Print the archEnemies array to console to ensure you added a new archEnemy
-
+console.log(archEnemies);
 // Remove the first sidekick from the sidekicks array
-
+sideKick.shift();
 // Print the sidekicks array to console to ensure you added a new sidekick
-
+console.log(sideKick);
 // Create a function called assessSituation that takes three arguments - dangerLevel, saveTheDay, badExcuse
 //   - dangerLevel should be an integer
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
+var saveTheDay = "All done here, for now!";
+var badExcuse = "I don't have enough strength for that";
```

Comment:
You don't need to declare `saveTheDay` and `badExcuse` as variables - they're just parameters in the function - essentially placeholders for whatever arguments get passed in when the function is called or invoked.